pkg/clusteragent/orchestrator/transformer.go

```diff
@@ -194,3 +197,174 @@ func extractDeploymentConditionMessage(conditions []v1.DeploymentCondition) stri
 	}
 	return ""
 }
+
+func extractNode(n *corev1.Node) *model.Node {
+	msg := &model.Node{
+		Metadata:      orchestrator.ExtractMetadata(&n.ObjectMeta),
+		PodCIDR:       n.Spec.PodCIDR,
+		PodCIDRs:      n.Spec.PodCIDRs,
+		ProviderID:    n.Spec.ProviderID,
+		Unschedulable: n.Spec.Unschedulable,
+		Status: &model.NodeStatus{
+			Allocatable: map[string]int64{},
+			Capacity:    map[string]int64{},
+		},
+	}
+
+	if n.Spec.Taints != nil && len(n.Spec.Taints) != 0 {
+		msg.Taints = extractTaints(n.Spec.Taints)
+	}
+
+	// status
+	extractNodeInfo(n, msg)
+	extractCapacitiesAndAllocatables(n, msg)
+
+	// extract status addresses
+	if n.Status.Addresses != nil && len(n.Status.Addresses) > 0 {
+		msg.Status.NodeAddresses = map[string]string{}
+		for _, address := range n.Status.Addresses {
+			msg.Status.NodeAddresses[string(address.Type)] = address.Address
+		}
+	}
+
+	// extract conditions
+	if n.Status.Conditions != nil && len(n.Status.Conditions) > 0 {
+		for _, condition := range n.Status.Conditions {
+			msg.Status.Conditions = append(msg.Status.Conditions, &model.NodeCondition{
+				Type:               string(condition.Type),
+				Status:             string(condition.Status),
+				LastTransitionTime: condition.LastTransitionTime.Unix(),
+				Reason:             condition.Reason,
+				Message:            condition.Message,
+			})
+		}
+	}
+
+	// extract status message
+	msg.Status.Status = computeNodeStatus(n)
```

Comment:
I'm wondering if we want this to be added as a tag (`node_status`) like we did for `pod_status` and if that should be the joined status or a tag with two values.

Reply:
I agree, status are nice information for filtering/grouping. 

My thought would be to do it similar to here: https://github.com/DataDog/datadog-agent/blob/cba028e7a8dfbc0edc5cbd80e1bab275c891fc38/pkg/process/util/orchestrator/pod.go#L68

Reply:
As we have no tagger running, I am thinking whether we want to do this on our backend or on the agent 🤔

Reply:
- Ready,unescheduled  vs
- Ready and Unschedulable 

I would be for option 2:
so:
`node_status:ready` and `node_status:unschedulable`